examples/redis/v1beta3/redis-slave.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1beta3
+kind: Pod
```

Comment:
We should delete this file. They should be using the replication controller now.